src/Validate.hs

```diff
@@ -52,6 +52,8 @@ validateMembers typeEnv typeVariables membersXObjs =
         Left var -> Left (InconsistentKinds var membersXObjs)
         _ -> pure ()
       where
+        -- fromJust is safe here; invalid types will be caught in the prior check.
+        -- todo? be safer anyway?
```

Comment:
I think it’d be good to be safe just in case anyone thinks of refactoring this in the future.

Reply:
Agreed! But let's merge this anyways, right?

Reply:
I think this can be merged unless @scolsen wants to add this to this PR!

Reply:
Let's merge as-is if that's ok. Fixing it in a safe way will require changing the signature of `checkMembers` and the functions it calls ever so slightly which is a bit more involved since we'd need to update call sites, we could also re-evaluate the same check in the body of `checkKinds` but that seems silly.

Reply:
Seems good to me!